build/ci/create_demo_pr.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -xe
+set +o pipefail
+
+yq -y --arg operator_image $operator_image_for_test '.spec.install.spec.deployments[0].spec.template.spec.containers[0].image |= $operator_image' $csv_file > temp.yaml
+cat temp.yaml > $csv_file
+yq -y --arg operator_image $operator_image_for_test '.metadata.annotations.containerImage |= $operator_image' $csv_file > temp.yaml
+cat temp.yaml > $csv_file
```

Comment:
I don't think we need to cat every time

Reply:
when I tried to put the input of the command directly to the `$csv_file`, it became empty

Reply:
I meant that simply one `cat` is sufficient (after the yaml file is finally ready)